src/config.ts

```diff
@@ -18,11 +18,10 @@ export const config = () => {
         604800: 'Last 7 days',
         2592000: 'Last 30 days'
       },
-      defaultPollInterval: 5000,
+      defaultPollInterval: 15000,
```

Comment:
I agree about increasing the default, but why removing shorter interval options? While monitoring metrics we sometimes want to have a shorter refresh rate. For instance Grafana shortest is 5s .

Reply:
Istio's Prometheus scrape interval is 15s. We'd essentially be making about 2 automatic calls unnecessarily to the backend server ("unnecessary" because we know the data isn't going to change until every 3rd call). To avoid stressing the server with those requests that we know will not provide new data, we limit it to the scrape interval.

Reply:
Good point... but scrape interval can still be configured.

Reply:
Configurable by Prometheus, yes, but AFAIK, it is not configurable from Istio (I don't know of a way to configure that from, say, the Istio install methods short of a user going in and manually modifying the ansible/helm scripts OR going in and somehow manually modifying the Prometheus configuration files when already installed - in either case, I don't see many people doing that - certainly the vast majority of Istio users won't be changing the scrape interval).

IMO, I do not believe we should give people the option to auto-refresh any lower than then default scrape interval when, for the vast majority of users, those lower refresh rates would be detrimental to the system performance (by hitting the server with 2 unnecessary requests) while providing no useful value, especially when there is no documented (and presumably supported) way to lower the scrape interval.

Reply:
Fair enough :)

Reply:
Actually... I was wondering why I was seeing updates every 5 seconds on the metrics screen. Turns out the default scrape interval is 5s, not 15: https://github.com/istio/istio/blob/2ac5ccdaa3fcd5dda5b3874b81e7ed7f33ffb80f/install/kubernetes/helm/istio/charts/prometheus/templates/configmap.yaml#L17-L33

Reply:
Interesting. That is the new helm stuff, so it looks like installing on Kubernetes with Helm will set it to 5. Do we know if the same is done when using the ansible installer on OpenShift?

Reply:
Looking at my OpenShift installed via Ansible installer, I also see this 5s - so it looks like they do override that 15s:

```
global:
  scrape_interval: 15s
  scrape_timeout: 10s
  evaluation_interval: 1m
scrape_configs:
- job_name: istio-mesh
  scrape_interval: 5s
  scrape_timeout: 5s
```

So this means we should put "5s" as an option in the refresh drop down... though I would agree that we should still default to something slower, like 15s or 30s. (this PR has it defaulting to 15s which is fine by me).

Reply:
Will add back the 5s but keep the default on 15s